tools/run_tests/run_interop_tests.py

```diff
@@ -536,9 +536,9 @@ def build_interop_image_jobspec(language, tag=None):
   # This env variable is used to get around the github rate limit
   # error when running the PHP `composer install` command
   # TODO(stanleycheung): find a more elegant way to do this
-  if language.safename == 'php' and os.path.exists('/var/local/.composer/auth.json'):
+  if language.safename == 'php' and os.path.exists('/home/jenkins/.composer/auth.json'):
```

Comment:
could we at least use current user's home dir instead of hardcoding /home/jenkins? That way the workaround can work for folks running the interop tests with docker locally.